docs/validate.rst

```diff
@@ -0,0 +1,39 @@
+Brick Validate
```

Comment:
Can you add some detail to these docs that describe the kinds of shapes that are bundled with the implementation?

Reply:
Sure.